documentation/asciidoc/computers/config_txt/autoboot.adoc

```diff
@@ -24,18 +24,50 @@ Set this property to `1` to load the normal `config.txt` and `boot.img` files in
 This enables the `tryboot` switch to be made at the partition level rather than the file-level without having to modify configuration files in the A/B partitions.
 
 
-=== A/B boot example
-In the following example partition `3` would be contain the pending OS upgrade and would be tested by rebooting in `tryboot` mode (`sudo reboot "0 tryboot"`).
-If the OS determines that the upgrade was successful then it would replace `autoboot.txt` swapping the partition numbers.
-Otherwise, if the system is reboot (e.g. watchdog or cold boot) then system would boot from partition `2` as usual.
+=== Example update flow for A/B booting
 
-Since the `autoboot.txt` file is a single sector it will normally be possible to update this with a single sector update to the SD/EMMC.
+The following pseudo code shows how an hypothetical OS `Update Serivce` could use `tryboot` + `autoboot.txt` to perform an fail-safe OS upgrade.t
```

Comment:
Typo. Serivce

Reply:
Done + fixed other typo for Update Service